run-build.ps1

```diff
@@ -30,6 +30,7 @@ if($Help)
 $env:CONFIGURATION = $Configuration;
```

Comment:
That's correct there isn't currently a bootstrap.sh and and I'm not aware of a tracking issue but I know it was on Joel's list in the past. It is definitely something we need.